Docs/Book/RDKit_Book.rst

```diff
@@ -2701,6 +2701,206 @@ path) enumeration algorithm used in the RDKit fingerprint. After a subgraph has
 been generated, it is used to set multiple bits based on different atom and bond
 type definitions.
 
+Self-Contained Structure Representations (SCSR) for Macromolecules
+*******************
+
+The SCSR support tentative added to RDKit follows the description in the BIOVIA document “biovia_ctfileformats_2020.pdf” available from 
+“CTfile Formats - Dassault Systèmes”.  That document does not provide any real detail for that format, and contains one example file.
+
+In addition, Biovia Draw supports reading and writing this format.  As much as possible, the RDKit support allows the functionality
+supported by Biovia Draw.  One exception is the RDkit treatment of hydrogen bonds in SCSR files/blocks (vide infra).
+
+
+Representation
+==============
+
+Main mol
+--------
+An SCSR file contains a mol with a CTAB in v3000 format.   That CTAB can contain elemental atoms and macro atoms corresponding to amino acids (AA), RNA, and DNA elements.  
+The RNA and DNA elements are represented by three parts – a phosphate group, a sugar, and a base.  
+
+Each atom line in the CAT can refer to an elemental atom or a macro atom.  Macro atom lines have a text description of the macro name and must have a CLASS and an
+ATTCHORD attributed.  They can also have an optional SEQID attribute.  
+
+According to the Biovia doc, the CLASS attribute must have a value that is one of:  AA, dAA, ScsrSupportInRDKitDNA, RNA, SUGAR, BASE, PHOSPHATE, LINKER, CHEM, LGR,P MODAA, MODdAA, MODDNA, MODRNA, XLINKAA, XLINKdAA, XLINKDNA, XLINKRNA.   
+This implementation in RDKit does NOT enforce the allowed list.
+
+The SEQID is a sequential integer and is ignored by this treatment.  Typically, the three parts of an RNA or DNA element have the same SEQID.
+
+The ATTCHORD attribute must have a specification for each bond that comes from the macro atom.   The specification is contained between parentheses, and the first 
+character is a integer that indicates the number of items that follow.  Each connection is specified with the atom ID (1 offset) for the connected atom in the main
+CTAB, and a string that indicates the attachment point for this macro atom.   The attachment point string is always two characters.  
+The first indicates the order, and is a capital letter starting at A.  The second char is one of “l” (left), “r” (right), “x” (cross connections – e.g. for cysteine).  
+In this implementation, the second character can also be “h” for hydrogen bond.   
+Thus, the attach connections are almost always in the list: “Al”, “Br”, “Cx” or “Ch”.   For example:
+
+    # ATTCHORD=(6 15 Br 13 Al 21 Cx)
```

Comment:
It's not clear to me why this starts with a `#`.
In order to get these to render as code samples, which I think is what you want, you need to end the previous line with `::` (yes, RST is irritating).
So this becomes:
```suggestion
Thus, the attach connections are almost always in the list: “Al”, “Br”, “Cx” or “Ch”.   For example::

    ATTCHORD=(6 15 Br 13 Al 21 Cx)
```

Please update all the other examples too.

Reply:
OK